<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-QS8DG931" failures="1" tests="2" name="Error Validations Test" time="20.525" errors="0" timestamp="2025-09-03T17:20:20 IST">
  <testcase classname="raubin.tests.ErrorValidationsTest" name="LoginErrorValidation" time="0.55"/>
  <testcase classname="raubin.tests.ErrorValidationsTest" name="ProductErrorValidation" time="12.263">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at raubin.tests.ErrorValidationsTest.ProductErrorValidation(ErrorValidationsTest.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- ProductErrorValidation -->
</testsuite> <!-- Error Validations Test -->
